### DISCUSSION TITLE
Unable to set up GitLab Oauth

### DISCUSSION THREAD

**Discussion by RaylynYu** [requarks/wiki#4088] [Help / Questions]
Hello, 
I am quite new to all this and was wondering if someone could help me in setting up GitLab as an Authentication strategy. I have tried but have been getting this error:
![image](https://user-images.githubusercontent.com/71283216/122079980-e6d97c00-cdcb-11eb-82c0-bb06ce766294.png)
 
I am wondering what step or info I am missing to make this work. Thanks!

**System Info:**
Cent OS 7
Wikijs Version 2.5.201 
Node.js 14
PostgreSQL 13

This is what I’ve done:
**In GitLab**
Created a New Application for the wiki, which generated the Application ID and Secret.
Callback URL:
https://[My Wiki.js IP address]/login/[key or token number]/callback 
(The URL was copy and pasted from the Configuration Reference in Wiki.js)
![image](https://user-images.githubusercontent.com/71283216/122080250-24d6a000-cdcc-11eb-8ce3-97017fbe0f6a.png)

**In Wiki.js**
Added GitLab as a strategy in Authentication
Copy /pasted the Application Id and Secret in Strategy Configuration
Added the base url:
https:// [MyGitLab IP Address]

**Comment by fengerzh** [comment]
I tried, it works now.

**Reply by mckaygerhard** [reply]
still does not work.. what is the clear steps?
